proto/substrait/algebra.proto

```diff
@@ -136,14 +136,42 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message DelimeterSeparatedTextReadOptions {
```

Comment:
typo in the message name here and in file_format

Reply:
thanks, fixed